components/LanguageSwitcher/LanguageSwitcher.vue

```diff
@@ -57,37 +65,19 @@
             </div>
           </a>
         </div>
-        <ul
-          v-if="!option.target && showSubMenu"
-          class="language-switcher__dropdown-menu child"
-        >
-          <li
-            v-for="child in option.children"
-            :key="child.key"
-            :class="{
-              'language-switcher__dropdown-menu--active':
-                child.lang === currentLocaleCode,
-            }"
-          >
-            <div class="language-switcher__lang">
-              <a
-                :href="
-                  useGetAbsoluteUrlIfSwitchWebsite(
-                    child.target,
-                    child.isOnPixOrg
-                  )
-                "
-              >
-                {{ $t(child.name) }}
-              </a>
-            </div>
-          </li>
-        </ul>
+        <language-switcher-sub-menu
+          :show-sub-menu="showSubMenu"
+          :available-languages="option.children"
+          :target="option.target"
+          :is-on-pix-org="option.isOnPixOrg"
+          :current-locale-code="currentLocaleCode"
+          :selected-menu="selectedMenu"
+        />
       </li>
     </ul>
     <span class="separator" />
   </div>
-  <div v-else-if="showLanguageDropdown && type === 'only-text'">
+  <div v-else-if="type === 'only-text'">
```

Comment:
Finalement il faut le remettre, non ?
```suggestion
  <div v-else-if="showLanguageDropdown && type === 'only-text'">
```